api/v1alpha1/databasemonitoring_types.go

```diff
@@ -0,0 +1,50 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
```

Comment:
Let's maybe clean this up? (multiple occurences, search for "EDIT THIS FILE" maybe?)